stable/distribution/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # JFrog Distribution Chart Changelog
 All notable changes to this project chart be documented in this file.
 
+## [1.0.0] - Sep 17, 2018
+* True HA with distributor and Redis in their own StatefulSets and headless services
+* Redis StatefulSet now part of the main templates
+* New Redis version: 4.0.11
```

Comment:
@eldada Can you add a note for the backward compatibility/upgrade in README.md

Reply:
+1

Reply:
See new changes to README along with other tunings.